src/main/scala/org/constellation/infrastructure/endpoints/TransactionEndpoints.scala

```diff
@@ -68,12 +68,18 @@ class TransactionEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F]
 
   def peerEndpoints(transactionService: TransactionService[F], metrics: Metrics) =
     getTransactionEndpoint(transactionService) <+>
-      getBatchEndpoint(metrics, transactionService)
+      getBatchEndpoint(metrics, transactionService) <+>
+      getLastTransactionRefEndpoint(transactionService)
 
   private def getTransactionEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root / "transaction" / hash =>
       transactionService.lookup(hash).map(_.asJson).flatMap(Ok(_))
   }
+
+  private def getLastTransactionRefEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "transaction" / "last-ref" / address =>
```

Comment:
Maybe `"transaction" / "latest" / address` for consistency with latest snapshot endpoint from block explorer 🤔

Reply:
We are actually fetching transaction reference and not a transaction data itself.